tests/test_dataset.py

```diff
@@ -392,20 +455,20 @@ def test_graph_standardization_logic(self):
         dataset_test = GraphDataset(
             hdf5_path = hdf5_path,
             target='binary',
-            standardize=True,
             train=False,
             dataset_train=dataset_train
         )
-
+        # mean and devs should be None
         assert dataset_train.means == dataset_test.means
         assert dataset_train.devs == dataset_test.devs
+        assert dataset_train.means is None
+        assert dataset_train.devs is None
 
         # raise error if dataset_train is not provided
         with self.assertRaises(TypeError):
```

Comment:
This and the `with self.assertRaises(TypeError):` below (row 476) should be moved to another test, we can call it `test_logic_graphdataset` and we can place it above `test_standardization_logic_graphdataset`